language/es6/iterator/usages.js

```diff
@@ -29,11 +29,11 @@ describe('Iterator usages', () => {
     function iteratorFunction() {
       return {
         next: function() {
-          return {value: consumableUsers.nextUser, done: consumableUsers.anyLeft}
+          return {done: !consumableUsers.anyLeft, value: consumableUsers.nextUser}
         }
       }
     }
-    usersIterable = {};
+    usersIterable = { [Symbol.iterator]: () => iteratorFunction() };
```

Comment:
```suggestion
    usersIterable = { [Symbol.iterator]: iteratorFunction };
```
wrapper function not necessery.